azavea/pwd-fishway
# Remove package-lock.json

## Overview

We use Yarn, so this file is superfluous, and appears to have been unintentionally added as part of #22

## Testing Instructions

 * On the `develop` branch you will see the following warning when running `scripts/update`:
    ```
    warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
    ```
 * On this branch, you should not see this warning when running `scripts/update`
